wire-ios/Wire-iOS/Resources/Base.lproj/Accessibility.strings

```diff
@@ -233,11 +233,13 @@
 "conversationsList.filter_menu_options.favorites.description" = "Show all favorite conversations";
 "conversationsList.filter_menu_options.groups.description" = "Show all group conversations";
 "conversationsList.filter_menu_options.oneOnOne.description" = "Show all one on one conversations";
+"conversationsList.filter_menu_options.folders.description" = "Folders";
 
-"conversationsList.filter_menu_options.allConversations.selected.description" = "Show all conversations";
```

Comment:
This accessibility labels were incorrect according to the design so updated them here.